lib/datasource/docker/common.ts

```diff
@@ -511,3 +511,14 @@ export async function getLabels(
     return {};
   }
 }
+
+export function isECRMaxResultsError(err): boolean | undefined {
```

Comment:
I don't like the implicit any here. Can make this more type safe?
```suggestion
export function isECRMaxResultsError(err: HttpError): boolean | undefined {
```

Or `RequestError`, check exports from `util/http` folder.